crypto/modes/build.info

```diff
@@ -34,7 +34,7 @@ IF[{- !$disabled{asm} -}]
 
   $MODESASM_ppc32=ghashp8-ppc.s
   $MODESDEF_ppc32=
-  $MODESASM_ppc64=$MODESASM_ppc32
+  $MODESASM_ppc64=$MODESASM_ppc32 aes-gcm-ppc.s
```

Comment:
The C code and the assembler sources look like this change should apply to ppc 32 bit builds as well but the assembler code is not added to ppc32 builds here.